app.js

```diff
@@ -38,29 +38,47 @@ var sessionSecret = process.env.SESSION_SECRET || settings.secret;
 var db = mongoose.connection;
 var dbOptions = { server: { socketOptions: { keepAlive: 1 } } };
 
+var fs = require('fs');
+var http = require('http');
+var https = require('https');
+var sslOptions = null;
+var server = http.createServer(app);
+var secureServer = null;
+
 app.set('port', process.env.PORT || 8080);
+app.set('securePort', process.env.SECURE_PORT || null);
 
 // Connect to the database
 mongoose.connect(connectStr, dbOptions);
 db.on('error', console.error.bind(console, 'connection error:'));
-db.once('open', function () {
-  app.listen(app.get('port'));
-});
+db.once('open', function () {});
 
 var sessionStore = new MongoStore({ mongooseConnection: db });
 
 // Force HTTPS
-if (app.get('port') === 443) {
+if (app.get('securePort')) {
```

Comment:
@sizzlemctwizzle 
So you are doing this programmatically now from the environment now instead of what was done at https://github.com/OpenUserJs/OpenUserJS.org/commit/932d1c906d70d6b305231d1ad381f60a7a717ea3 ? e.g. which no longer allows http://openuserjs.org ?